src/Storages/Distributed/DirectoryMonitor.cpp

```diff
@@ -623,6 +629,12 @@ struct StorageDistributedDirectoryMonitor::Batch
 
         CurrentMetrics::Increment metric_increment{CurrentMetrics::DistributedSend};
 
+        Stopwatch watch;
+
+        LOG_TRACE(parent.log, "Sending a batch of {} files ({} rows, {} bytes).", file_indices.size(),
+            formatReadableQuantity(total_rows),
```

Comment:
See also `ReadableSize` proxy object.

Reply:
Right, but it works only for `formatReadableSizeWithBinarySuffix`, and I don't see how wrapping with `ReadableSize` better then explicit `formatReadableSizeWithBinarySuffix` (since it still requires wrapping into `ReadableSize` object), is using `ReadableSize` preferable?

Reply:
Does not matter.